eslint.config.mjs

```diff
@@ -0,0 +1,27 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+
+
+/** @type {import('eslint').Linter.Config[]} */
+export default [
+  pluginJs.configs.recommended, 
+  {
+    files: ["**/*.js"], 
+    languageOptions: {
+      sourceType: "commonjs",
+      globals: globals.browser
+    },
+    rules: {
+      "no-undef": "off",
+      "no-unused-vars": "off",
+      "no-prototype-builtins": "off",
+      "no-redeclare": "off",
+      "no-useless-escape": "off",
+      "no-unexpected-multiline": "off",
+      "no-cond-assign": "off",
+      "require-yield": "off",
+      // "camelcase": "error",
```

Comment:
If we were to add lint, why disable all those errors?

Reply:
Those errors are on in default linter options, but they are violated in Datastax code. I planned to add an issue for investigating and fixing them.

Reply:
In this case I would prefer to have those changes in this PR